knowledge-base/kb-filedialogsdemos-notavailable.md

```diff
@@ -0,0 +1,37 @@
+---
+title: FileDialogs Demos Not Available in WPF Control Examples App
+description: "How to run FileDialogs WPF Demos application." 
+type: how-to
+page_title: How to run FileDialogs WPF Demos.
+slug: kb-filedialogsdemos-notavailable
+position: 0
+tags: filedialogs, demos
+ticketid: 1615001
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadFileDialogs for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to run `FileDialogs Demos` in `WPF Control Examples` application. 
+This often happens when you have an older installation of the `WPF Demos` application and you install the latest one. It is related to cache in the [ClickOnce]({%kb-sdk-samples-browser-cannot-be-installed-or-run%}) files of the application.
```

Comment:
Add a picture with the issue. You can use the one from the ticket.

To add a picture use the following syntax `![WPF RadHeatMap Overview](images/radheatmap-overview-0.png)`

https://www.markdownguide.org/basic-syntax/#images-1